Framework/include/StopJets.h

```diff
@@ -69,16 +65,15 @@ class StopJets
         std::vector<TLorentzVector> notTopJets;
         for(unsigned int i = 0; i < Jets.size(); ++i)
         {
-            if(!GoodJets_pt45[i]) continue;
+            if(!GoodJets_pt20[i]) continue;
             if ( std::find(usedIndex.begin(), usedIndex.end(), i) == usedIndex.end() ) 
             {
-                notTopJets.push_back(Jets[i]);
+                StopJets_.push_back(Jets[i]);
             }
         }
     
         // get the StopJets : add tops and not tops jets to each other
-        auto& StopJets = tr.createDerivedVec<TLorentzVector>("StopJets"+myVarSuffix_, topJets);
-        StopJets.insert(StopJets.end(), notTopJets.begin(), notTopJets.end());
+        auto& StopJets = tr.createDerivedVec<TLorentzVector>("StopJets"+myVarSuffix_, StopJets_);
```

Comment:
Move this line to line 33 and remove the second argument. With createDerivedVec you make a vector then fill it.